### PR TITLE
[browser][MT] disable jiterp + enable safe points

### PR DESCRIPTION
**Pull request by pavelsavara**
In hope that it will make https://github.com/dotnet/runtime/issues/99888 go away

### CODE REVIEW

File: src/mono/mono/utils/options-def.h [hunk 1]
```diff
@@ -81,7 +80,7 @@ DEFINE_BOOL(jiterpreter_interp_entry_enabled, "jiterpreter-interp-entry-enabled"
 DEFINE_BOOL(jiterpreter_jit_call_enabled, "jiterpreter-jit-call-enabled", TRUE, "JIT specialized WASM do_jit_call trampolines")
 #else
 // traces_enabled controls whether the jiterpreter will JIT individual interpreter opcode traces
-DEFINE_BOOL(jiterpreter_traces_enabled, "jiterpreter-traces-enabled", TRUE, "JIT interpreter opcode traces into WASM")
+DEFINE_BOOL(jiterpreter_traces_enabled, "jiterpreter-traces-enabled", FALSE, "JIT interpreter opcode traces into WASM")
```
Comment (kg) [inline src/mono/mono/utils/options-def.h:83]: You should make this DEFINE_BOOL_READONLY to *ensure* that it's not on even if someone tries to manually enable it

**Review by maraf — APPROVED** [review]

**Comment by pavelsavara** [comment]
I filed https://github.com/dotnet/runtime/issues/100237

**Review by kg — APPROVED** [review]

**Comment by pavelsavara** [comment]
> In hope that it will make #99888 go away

It didn't fix it. I created https://github.com/dotnet/runtime/issues/100273 to enable it back in the future.

**Pull request merged**